README.adoc

```diff
@@ -0,0 +1,114 @@
+= Agriboard
+
+Agriboard is an administrative dashboard for the agriculture software stack.

```

Comment:
How did this get added. Where did my commits go to add this before lol. @Right2Drive

Reply:
Umm would this by any chance be tied to the fact that when I first tried to do the pull request, I had the ‘source’ and ‘destination’ branches backwards? I closed it, then re-made the pull request, this time with the branches in the correct order.